diff -u a/src/scanner.cc b/src/scanner.cc
--- a/src/scanner.cc	2023-02-03 11:48:23.490095119 +0100
+++ b/scanner.cc	2023-02-03 11:41:46.390092778 +0100
@@ -82,6 +82,10 @@
   string scan_tag_name(TSLexer *lexer) {
     string tag_name;
     while (iswalnum(lexer->lookahead) ||
+	   // to accept xml names!
+           lexer->lookahead == '.' ||
+           lexer->lookahead == '_' ||
+	   // original html-only name
            lexer->lookahead == '-' ||
            lexer->lookahead == ':') {
       tag_name += towupper(lexer->lookahead);

Diff finished.  Fri Feb  3 11:48:46 2023
